proto/server/main.cc

```diff
@@ -0,0 +1,8 @@
+#include "server.h"
+
+#include <string>
+
+int main(){
+  RunServer("0.0.0.0", "1234");
+  return 0;
+}
```

Comment:
Let's just put the main function in server.cc.

Reply:
Done